normandy/base/api/mixins.py

```diff
@@ -0,0 +1,14 @@
+from django.conf import settings
+from django.views.decorators.cache import cache_control
+
+
+class CachingMixin:
```

Comment:
I can't remember what we decided RE: subclassing `object` or not. A lot of our test classes do, for example, as do some middlewares and other classes around the codebase I found do.